changelog/unreleased/pull-2733

```diff
@@ -0,0 +1,7 @@
+Enhancement: Enable support for EKS IRSA
```

Comment:
Actually this PR updates the minio library used for S3-like back-ends. And as a side-effect we get support for AWS EKS IRSA (hmm, that are too many abbreviations at once).

I would suggest to call this `S3 backend: Add support for WebIdentityTokenFile`.